components/blocks/CompareBox/CompareBox.template.ts

```diff
@@ -1,79 +1,148 @@
-import type { Template } from 'tinacms';
+import { useEffect, useState } from "react";
+import { wrapFieldsWithMeta, type Template } from "tinacms";
+import { checkboxList } from "../../../tina/customTinaFormFields/checkboxList";
+
+export const splitOneAndJoin = (item, separator) => {
```

Comment:
add comment for what first is and what ...rest is 

(i.e from our convo - first is the boolean flag, ...rest is... well everything else)